tests/sandbox/test_jax.py

```diff
@@ -222,6 +222,19 @@ def test_jax_basic():
     )
 
 
+@pytest.mark.skip("Currently in WIP and broken")
+def test_jax_eye():
+    """Tests that jaxification of eye operator
+
+    Currently WIP
+    """
+    x = tt.matrix("x")
+    out = tt.eye(3)
+    out_fg = theano.gof.FunctionGraph(x, [out])
```

Comment:
If this `FunctionGraph` doesn't need any inputs (i.e. `x`), you can just give it an empty list.

Also, try using symbolic inputs as the arguments for `tt.eye` (e.g. `x, y, z = tt.iscalars(3); out = tt.eye(x, y, z)`).